src/Entity/User.php

```diff
@@ -19,15 +21,19 @@
         new GetCollection(),
         new Post(),
         new Put(),
-        new Delete(),
-    ]
+    ],
+    formats: ['json'],
+    collectDenormalizationErrors: true
 )]
+#[ApiFilter(filterClass: SearchFilter::class, properties: ['name' => 'partial', 'isActive' => 'exact'])]
```

Comment:
### ❗ Issue: Pls add `member_uid` to this list

We should have an opportunity to get all `members` of some `event`